31Aug_MajorityElement.py

```diff
@@ -0,0 +1,33 @@
+# https://leetcode.com/problems/majority-element/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+#####################################################################################################################################################
+#
+# Given an array nums of size n, return the majority element.
+# The majority element is the element that appears more than ⌊n / 2⌋ times. You may assume that the majority element always exists in the array.
+#
+# Example 1:
+# Input: nums = [3,2,3]
+# Output: 3
+#
+# Example 2:
+# Input: nums = [2,2,1,1,1,2,2]
+# Output: 2
+#
+class Solution:
+    def majorityElement(self, nums: List[int]) -> int:
+      # nums = [3, 2, 3]
+      a = Counter(nums)
+      # a = Counter({3: 2, 2: 1})
+      max_value = 0
+      result = 0
+      for i, j in a.items():
```

Comment:
Normally dummy names are fine for tight loops, but in this case I would probably could for `number, count` to make it clearer

Reply:
```
      max_value = 0
      result = 0
      for number, count in total.items():
```